src/main/java/uk/gov/hmcts/probate/model/ccd/raw/request/CaseDataParent.java

```diff
@@ -4,21 +4,73 @@
 import lombok.Getter;
 import lombok.experimental.SuperBuilder;
 import lombok.extern.jackson.Jacksonized;
+import uk.gov.hmcts.probate.model.ccd.raw.AdditionalExecutorNotApplyingPowerReserved;
+import uk.gov.hmcts.probate.model.ccd.raw.AdditionalExecutorPartners;
+import uk.gov.hmcts.probate.model.ccd.raw.AdditionalExecutorTrustCorps;
+import uk.gov.hmcts.probate.model.ccd.raw.CodicilAddedDate;
+import uk.gov.hmcts.probate.model.ccd.raw.CollectionMember;
+import uk.gov.hmcts.probate.model.ccd.raw.SolsAddress;
 
 import java.time.LocalDate;
+import java.util.List;
 
 @Jacksonized
 @SuperBuilder
 @Data
 public class CaseDataParent {
 
-    protected String registrySequenceNumber;
+    protected final String schemaVersion;
+    protected final String registrySequenceNumber;
     protected final String deceasedDeathCertificate;
     protected final String deceasedDiedEngOrWales;
     protected final String deceasedForeignDeathCertInEnglish;
     protected final String deceasedForeignDeathCertTranslation;
+    protected final String solsForenames;
+    protected final String solsSurname;
+    protected final String solsSolicitorWillSignSOT;
+    protected final String dispenseWithNotice;
+    protected final String titleAndClearingType;
+    protected final String trustCorpName;
+    protected SolsAddress trustCorpAddress;
+    // Not final so field can be reset in CaseDataTransformer
```

Comment:
Do we need to keep these comments , think should be removed

Reply:
Yes happy to if you'd like me to, previously CaseData/CaseDataParent was immutable, but we needed to allow it mutate for TrustCorps to simplify executor mapping (it's still complex and could do with a refactor, but at least its not as complex as it would have been if we couldn't mutate the request data). So this just explains why we had to make mutable. Happy to remove tho